classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
```

Comment:
- [ ] The `@students` variable should only have a getter method. It should only be read from outside the class and not set to a new value from outside the class. Kindly implement this change 👍 💻 

`attr_reader :students`